src/librustc_target/lib.rs

```diff
@@ -17,3 +17,8 @@
 
 pub mod abi;
 pub mod spec;
+
+/// Requirements for a `StableHashingContext` to be used in this crate.
+/// This is a hack to allow using the `HashStable_Generic` derive macro
+/// instead of implementing everything in librustc.
+pub trait StableHashingContextLike {}
```

Comment:
I'd like this to be named `HashStableContext`.

Reply:
I see you named this `StableHashingContext` in your commits. There's already a type with that name though and trait are preferred to be in the verb form in Rust.

Reply:
Indeed, I misread your comment.